src/vs/workbench/contrib/debug/browser/repl.ts

```diff
@@ -237,7 +237,9 @@ export class Repl extends FilterViewPane implements IHistoryNavigationWidget {
 			this.filter.filterQuery = this.filterWidget.getFilterText();
```

Comment:
I don't totally follow this, is it right that this event is for when the (old) filter widget text changes? But you're typing in the new one?

Reply:
this didn't actually fix the bug 🤦🏼‍♀️ . I just wasn't reproducing it. This should be reverted - it's not causing issues though, so will wait on the real fix https://github.com/microsoft/vscode/pull/225300